test/relayclient/RelayClient.test.ts

```diff
@@ -211,7 +223,8 @@ contract('RelayClient', function (accounts) {
 
       // @ts-ignore (sinon allows spying on all methods of the object, but TypeScript does not seem to know that)
       sinon.spy(dependencyTree.knownRelaysManager)
-      await relayClient._attemptRelay(relayInfo, optionsWithGas)
+      const attempt = await relayClient._attemptRelay(relayInfo, optionsWithGas)
+      assert.equal(attempt.error?.message, 'some error describing how timeout occurred somewhere')
```

Comment:
huh? where did this error came from?

Reply:
This is from a stubbed method, don't worry!